tutorials/llama_index_multimodal_image_reasoning.ipynb

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
<p>This feels like the title and instructions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Its meant to be hehe</p>